codespell_lib/data/dictionary.txt

```diff
@@ -24445,6 +24452,7 @@ skeep->skip
 skelton->skeleton
 skept->skipped
 sketchs->sketches
+skilfull->skillfull, skilful,
```

Comment:
Cambridge has your version as wrong, should be:
```suggestion
skilfull->skillful, skilful,
```
https://dictionary.cambridge.org/dictionary/english/skilful

It would be therefore good to add your mistake and the other variants.

Reply:
damn. i hate this word. :-D

Reply:
added